controllers/clustercache/cluster_cache.go

```diff
@@ -735,5 +753,6 @@ func buildClusterAccessorConfig(scheme *runtime.Scheme, options Options, control
 			Interval:         10 * time.Second,
 			FailureThreshold: 5,
 		},
+		ClusterFilter: options.ClusterFilter,
```

Comment:
I think we should store the filter on the clustercache not on the clusterAccessor